src/main/java/de/tum/in/www1/artemis/web/rest/DataExportResource.java

```diff
@@ -11,8 +11,7 @@
 
 import org.springframework.beans.factory.annotation.Value;
 import org.springframework.core.io.Resource;
-import org.springframework.http.MediaType;
-import org.springframework.http.ResponseEntity;
+import org.springframework.http.*;
```

Comment:
This should not be a wildcard import (unless I miscounted)

Reply:
I counted 4 occurrences which is consistent with our .editorconfig